pyaedt/icepak.py

```diff
@@ -4869,3 +4869,341 @@ def assign_symmetry_wall(
+            - For the ``"Function"`` key, acceptable values are
+            ``"Exponential"``, ``"Linear"``, ``"Piecewise Linear"``,
+            ``"Power Law"``, ``"Sinusoidal"``, and ``"Square Wave"``.
+            - For the ``"Values"`` key, a list of strings containing the
+            parameters required by the ``"Function"`` key selection. For
+            example, when``"Linear"`` is set as the ``"Function"`` key, two
+            parameters are required: the value of the variable at t=0 and the
+            slope of the line. For the parameters required by each
+            ``"Function"`` key selection, see the Icepak documentation.
+            The parameters must contain the units where needed.
+            The default value is ``"0cel"``.
+        conductance_external_temperature : str, optional
+            External temperature value, needed if ``thermal_specification``
+            is set to ``"Conductance"``. Default is ``None``.
+        flow_specification: str, optional
+            Flow specification for the recirculation zone. Available
+            options are: ``"Mass Flow"``, ``"Mass Flux"``, and
+            ``"Volume Flow"``. The default value is ``"Mass Flow"``.
+        flow_assignment: str or dict, optional
+            String with value and units of the flow assignment. For a
+            transient assignment, a dictionary can be used. The dictionary
+            should contain two keys: ``"Function"`` and ``"Values"``.
+            - For the ``"Function"`` key, acceptable values are
+            ``"Exponential"``, ``"Linear"``, ``"Piecewise Linear"``,
+            ``"Power Law"``, ``"Sinusoidal"``, and ``"Square Wave"``.
+            - For the ``"Values"`` key, a list of strings containing the
+            parameters required by the ``"Function"`` key selection. For
+            example, when``"Linear"`` is set as the ``"Function"`` key, two
+            parameters are required: the value of the variable at t=0 and the
+            slope of the line. For the parameters required by each
+            ``"Function"`` key selection, see the Icepak documentation.
+            The parameters must contain the units where needed.
+            The default value is ``"0kg_per_s_m2"``.
+        flow_direction : list, optional
+            Flow direction enforced at the recirculation zone. The default value
+            is ``None`` in which case the normal direction is used.
+        start_time : str, optional
+            Start of the time interval. Relevant only if the simulation is
+            transient. The default value is ``"0s"``.
+        end_time : str, optional
+            End of the time interval. Relevant only if the simulation is
+            transient. The default value is ``"0s"``.
+        boundary_name : str, optional
+            Name of the recirculation boundary. The default is ``None``, in
+            which case the boundary is automatically generated.
```

Comment:
```suggestion
            String with the value and units of the flow assignment. For a
            transient assignment, a dictionary can be used. The dictionary
            should contain two keys: ``"Function"`` and ``"Values"``.

            - For the ``"Function"`` key, options are
            ``"Exponential"``, ``"Linear"``, ``"Piecewise Linear"``,
            ``"Power Law"``, ``"Sinusoidal"``, and ``"Square Wave"``.
            - For the ``"Values"`` key, provide a list of strings containing the
            parameters required by the ``"Function"`` key selection. For
            example, when``"Linear"`` is set as the ``"Function"`` key, two
            parameters are required: the value of the variable at t=0 and the
            slope of the line. For the parameters required by each
            ``"Function"`` key selection, see the Icepak documentation.
            The parameters must contain the units where needed.
            The default value is ``"0kg_per_s_m2"``.
        flow_direction : list, optional
            Flow direction enforced at the recirculation zone. The default value
            is ``None``, in which case the normal direction is used.
        start_time : str, optional
            Start of the time interval. This parameter is relevant only if the simulation is
            transient. The default value is ``"0s"``.
        end_time : str, optional
            End of the time interval. This parameter is relevant only if the simulation is
            transient. The default value is ``"0s"``.
        boundary_name : str, optional
            Name of the recirculation boundary. The default is ``None``, in
            which case the boundary is automatically generated.
```
For the second parameter, I have the same questions about the default value of ``"0kg_per_s_m2"``.  I'm not sure what the default value is for as it seems that the values for the "Function" key selection can vary.